pkg/consts/consts.go

```diff
@@ -47,6 +48,7 @@ const (
 )
 
 // These consts are used to query Repository server API objects
+
```

Comment:
is this required? i would prefer is removed.

Reply:
refactored.